packages/backend/src/app/ee/pieces/platform-piece-module.ts

```diff
@@ -0,0 +1,54 @@
+import { FastifyPluginAsyncTypebox } from '@fastify/type-provider-typebox'
+import { FastifyPluginCallbackTypebox } from '@fastify/type-provider-typebox'
+import { ActivepiecesError, AddPieceRequestBody, ErrorCode, isNil } from '@activepieces/shared'
+import { PieceMetadataModel } from '../../pieces/piece-metadata-entity'
+import { platformService } from '../platform/platform.service'
+import { pieceService } from '../../pieces/piece-service'
+
+
+export const platformPieceModule: FastifyPluginAsyncTypebox = async (app) => {
+    await app.register(platformPieceController, { prefix: '/v1/pieces' })
+}
+
+const platformPieceController: FastifyPluginCallbackTypebox = (app, _opts, done) => {
+    app.post('/', {
+        schema: {
+            body: AddPieceRequestBody,
+        },
+    }, async (req): Promise<PieceMetadataModel> => {
+        const { platformId } = req.body
+        const { packageType, pieceName, pieceVersion, pieceArchive } = req.body
+        const { projectId } = req.principal
+        await assertUserIsPlatformOwner({
+            platformId,
+            userId: req.principal.id,
+        })
+        return await pieceService.add({
```

Comment:
is it `pieceMetadataService#create` or `pieceService.add`?
(blocking)

Reply:
pieceService#add similar to community piece module

Reply:
Renamed to Install Piece